pupil/test/config/browserstack.yml

```diff
@@ -0,0 +1,76 @@
+server: "hub-cloud.browserstack.com"
+user: mohsen20
+authkey: ZyZxbzNCWSdnp1sgLCEz
```

Comment:
Please revoke this key immediately.  Credentials can be read from the environment instead.